api/app/main.py

```
@@ -0,0 +1,710 @@
+		root.addHandler(h)
+	root.setLevel(logging.INFO)
+
+
+_configure_logging()
+
+from .db_monitor import SQLiteSwapMonitor
+from .models import Swap
+from .store import SwapStore
+from .events import load_events
+from .prices import CoinConfig, PriceCache
+import logging
```

Ruff (0.14.5)

Redefinition of unused `logging` from line 8

Remove definition: `logging`

(F811)